src/web/mjs/connectors/ManHuaGui.mjs

```diff
@@ -24,23 +22,7 @@ export default class ManHuaGui extends SinMH {
                 value: 2500
             }
         };
-    }
-
-    async _getMangas() {
-        let msg = 'This website does not provide a manga list, please copy and paste the URL containing the chapters directly from your browser into HakuNeko.';
-        throw new Error(msg);
-    }
-
-    async _getChapters(manga) {
-        let chapterList = await super._getChapters(manga);
-        return chapterList.map(chapter => {
-            chapter.title = chapter.title.replace(/\d+p$/, '');
-            return chapter;
-        });
-    }
-
-    async _getPages(chapter) {
-        let script = `
+        this.scriptPages =`
             new Promise((resolve, reject) => {
                 setTimeout(() => reject(new Error('Failed to get page links!')), 5000);
```

Comment:
This is redundant.
`fetchUI` provides an additional timeout parameter which defaults to 60 seconds, but can be changed.
That said, there is no reason for timeout at all as the promise will throw as soon as any error occurs in the script.